test/index.js

```diff
@@ -152,43 +160,36 @@ describe('preBuild()', () => {
 })
 
 describe('onBuild()', () => {
-  test('runs NoN with functions_src & publish_dir options', async () => {
-    const PUBLISH_DIR = 'some/path'
-    const FUNCTIONS_SRC = 'other/path'
-    await plugin.onBuild({
-      constants: {
-        PUBLISH_DIR,
-        FUNCTIONS_SRC,
-      },
-    })
-
-    const nextOnNetlifyOptions = nextOnNetlify.mock.calls[0][0]
-    expect(nextOnNetlifyOptions.functionsDir).toEqual(FUNCTIONS_SRC)
-    expect(nextOnNetlifyOptions.publishDir).toEqual(PUBLISH_DIR)
+  beforeEach(() => {
+    moveNextDist()
```

Comment:
Instead of `beforeEach()`, could we repeat `moveNextDist()` in the beginning of each test function, like we do with `useFixture()`? This would make it clearer when looking at a specific file that this step is performed first.